### PR TITLE
CE review - Getting started revamp Virtual device section

### PR DESCRIPTION
**Pull request by MuchoLucho**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [x] Make sure that all commits follow the conventional commit [specification](https://github.com/mendersoftware/mendertesting/blob/master/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [x] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Revamp of Get started docs for Virtual devices and "Deploying an application Update" that is related to the "Virtual Device" section

### CODE REVIEW

**Comment by TheMeaningfulEngineer** [comment]
@merlin-northern 
Hey,
you're doing the onboarding test this sprint.
Could you follow the docs from this PR and do the review of these doc as part of it?

**Comment by merlin-northern** [comment]
> @merlin-northern Hey, you're doing the onboarding test this sprint. Could you follow the docs from this PR and do the review of these doc as part of it?

@TheMeaningfulEngineer any chance it will get merged this week?

File: 01.Get-started/02.Deploy-an-application-update/docs.md [hunk 1]
```diff
@@ -107,12 +112,12 @@ Leave the next prefilled fields as they are and click **UPLOAD ARTIFACT** as bel
...
 ![artifact details 2](artifact_details_2.png)
 
-You will now see a new Release containing this Artifact under the **Releases** tab. View its details, and create a deployment by clicking **Create a deployment for this Release** 
+You will now see a new Release containing this Artifact under the **Releases** tab. View its details, and create a deployment by clicking **Create a deployment for this Release**.
 
 ![view release](release_actions.png)
 
 Click through the deployment wizard to make a new deployment, similarly to what you did in
-[Step 1 - Create a Deployment](#step-1-create-a-deployment); this time, use the *hello-world* Release.
+[Step 1 - Create a Deployment](#step-1---create-a-deployment); this time, use the *hello-world* Release.
```
Comment (michaelntech) [inline 01.Get-started/02.Deploy-an-application-update/docs.md:120]: This adjusted link no longer works for me - the original `#step-1-create-a-deployment` works

**Review by michaelntech — CHANGES_REQUESTED** [review]
All looks good, except for the link I commented on!

**Review by TheMeaningfulEngineer — APPROVED** [review]

**Review by michaelntech — APPROVED** [review]

**Pull request merged**